crt_portal/static/sass/_uswds-theme-color.scss

```diff
@@ -24,57 +24,57 @@ Theme palette colors
 */
 
 // Base colors
-$theme-color-base-family:           'gray-cool';
```

Comment:
This file shouldn't be in here.  I think it's showing up because I based this on the previous color palette work.